src/loader.ts

```diff
@@ -0,0 +1,6 @@
+// Hook require() to transform to CJS
+require('@esbuild-kit/cjs-loader');
```

Comment:
I changed this to `require()` from `import`.

When using `import`, the ESM loader ends up calling require after determining it's a CJS package. Calling `require()` directly short-circuits it.